### DISCUSSION TITLE
Backup recommendation

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#2629] [Questions] [answered]
Why is the Backup recommendation only given, before updating the "service" containers, but not before updating the mastercontainer itself?

**Comment by szaimen** [comment, marked as answer]
Hey, so in general you can without problem create a backup before and after the mastercontainer update. However there have been cases in the past where it was better to create a backup after the mastercontainer update. So that is what is recommended in the AIO interface. Also to not increase the confusion when to create the backup is the warrning only shown at this specific step. Additionally, mastercontainer updates should work without any problem so a backup is not necessarily needed in contrast to Nextcloud updates imho.
